lib/quadquizaminos_web/views/page_view.ex

```diff
@@ -1,11 +1,31 @@
 defmodule QuadquizaminosWeb.PageView do
   use QuadquizaminosWeb, :view
   alias Quadquizaminos.Accounts
+  @bom_dir Application.app_dir(:quadquizaminos, "priv/static/.well-known")
 
   def render("instructions.html", _params) do
     QuadquizaminosWeb.Instructions.game_instruction()
   end
 
+  def render("sbom.html", _params) do
+    dir_files =
+      for dir <- @bom_dir |> File.ls!(),
```

Comment:
When the `/.well-known` directory is missing , this blows up. Will there be need to cater for such case?